buildSrc/src/main/java/AndroidComponents.kt

```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "48.0.20200624130112"
+    const val VERSION = "48.0.20200626130049"
```

Comment:
We could also use a slightly newer build 🤷  `48.0.20200626213814`

Reply:
I just updated it @ekager  you beat me to the punch 😆